src/modules/tests/tests.selector.ts

```diff
@@ -14,14 +14,18 @@ const outcomeStatus: any[] = [
   { outcomeCode: '5', outcomeText: 'Unsuccessful', outcomeColour: 'mes-red' },
 ];
 
+export const getCurrentTestSlotId = (tests: TestsModel) => {
+  return tests.currentTest.slotId;
+};
+
```

Comment:
can we shorten this?

`export const getCurrentTestSlotId = (tests: TestsModel) => tests.currentTest.slotId;`